enterprise/backend/test/metabase_enterprise/advanced_permissions/common_test.clj

```diff
@@ -12,6 +11,26 @@
             [metabase.test :as mt]
             [metabase.util :as u]))
 
+(defn- do-with-all-user-data-perms
+  [graph f]
+  (let [all-users-group-id  (u/the-id (group/all-users))
+        current-graph       (get-in (perms/data-perms-graph) [:groups all-users-group-id])]
+    (def my-current-graph current-graph)
+    (premium-features-test/with-premium-features #{:advanced-permissions}
+      (memoize/memo-clear! @#'field/cached-perms-object-set)
+      (try
+        (mt/with-model-cleanup [Permissions]
+          (@#'perms/update-group-permissions! all-users-group-id graph)
+          (f))
+        (finally
+          (@#'perms/update-group-permissions! 1 my-current-graph))))))
```

Comment:
```suggestion
          (@#'perms/update-group-permissions! all-users-group-id current-graph))))))
```

Reply:
Yeah it's definitely not a good idea to assume the All Users group is group 1. It might be a different group as a result of dump/load/serialization or something like that

Reply:
Sorry, these were left over from debugging. Was moving too fast 😢